test/checks/color/color-contrast.js

```diff
@@ -157,4 +157,39 @@ describe('color-contrast', function () {
 		assert.equal(checkContext._data.missingData[0].reason, 'bgGradient');
 		assert.equal(checkContext._data.contrastRatio, 0);
 	});
+
+	it('should return undefined when there are elements overlapping', function () {
+		fixture.innerHTML = '<div style="color: black; background-color: white; width: 200px; height: 100px; position: relative;" id="target">' +
+			'My text <div style="position: absolute; top:0; left: 0; background-color: white; width: 100%; height: 100%;"></div></div>';
+		var target = fixture.querySelector('#target');
+		assert.isUndefined(checks['color-contrast'].evaluate.call(checkContext, target));
+		assert.equal(checkContext._data.missingData[0].reason, 'bgOverlap');
+		assert.equal(checkContext._data.contrastRatio, 0);
+	});
+
+	it('should return true when a form wraps mixed content', function() {
```

Comment:
It was an extra check just to make sure the DOM walker didn't break in that case. We probably don't need it though